src/core/base.js

```diff
@@ -1,18 +1,20 @@
 'use strict';
-const _ = require('lodash');
+
 const sjcl = require('./utils').sjcl;
-const utils = require('./utils');
+const sha256 = require('./utils').sha256;
 const extend = require('extend');
-const convertBase = require('./baseconverter');
 
 const Base = {};
 
 const alphabets = Base.alphabets = {
+  bitcoin: '123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz',
   ripple: 'rpshnaf39wBUDNEGHJKLM4PQRST7VWXYZ2bcdeCg65jkm8oFqi1tuvAxyz',
-  tipple: 'RPShNAF39wBUDnEGHJKLM4pQrsT7VWXYZ2bcdeCg65jkm8ofqi1tuvaxyz',
-  bitcoin: '123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz'
+  tipple: 'RPShNAF39wBUDnEGHJKLM4pQrsT7VWXYZ2bcdeCg65jkm8ofqi1tuvaxyz'
 };
 
+const codecOptions = {sha256, alphabets, defaultAlphabet: 'ripple'};
```

Comment:
We could give the `alphabet` keyword argument actual alphabet semantics as you said, and call the symbolic name something else. `codec` shrug?

I think it's fine as it is. It's not like we're talking about an api that is used all the time, and the verbosity would lead to pain and disgrace. Each app would configure it exactly once, if that. 

As you said  `probably only going to use one of them and ripple-address-codec will already have it specified.`

ripple-client uses the symbolics, eg:  https://github.com/ripple/ripple-client/blob/develop/src/js/directives/validators.directive.js#L160